src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: { name, age, sex, partnerName, isMarried },
```

Comment:
Such destructuring should be done in the middle of the component, not in the arguments